fe/fe-core/src/main/java/org/apache/doris/analysis/CreateMaterializedViewStmt.java

```diff
@@ -30,11 +34,6 @@
 import org.apache.doris.common.FeNameFormat;
 import org.apache.doris.common.UserException;
 
```

Comment:
plz import the jar package according following sequence：
4=javax
3=java
2=org
1=com
0=org.apache.doris

Reply:
Hi, you could update the `checkstyle.xml`.

Reply:
Changed